### ISSUE TITLE
Add socket remote host and port to log messages

### ISSUE THREAD

**Issue by lillem4n** [larvit/larvitsmpp#8]
For traceability it would be very valuable. When several clients are connected the log gets spammed and it is hard to follow a single clients session.

**Comment by qasimakhan** [comment]
You can add client ID in the logs and filter them from there on...

**Comment by lillem4n** [comment]
Yes, that is an even better idea! I'd like client ID and at least the IP though, since many times when searching the logs you save some time by simply filter on the clients ID no need to look up the session ID. Especially irritating would be if the client and server keeps reconnecting so the ID changes all the time. :)

**Comment by lillem4n** [comment]
This will be implemented in [@larvit/smpp](https://www.npmjs.com/package/@larvit/smpp), the successor of larvitsmpp, in its repository on Gitea: https://gitea.larvit.se/larvit/smpp-js. As of 0.5.0 only `server - incoming connection` logs the remote address and port. Closing here.
